### DISCUSSION TITLE
New Example Project: riverpod_todo_with_changenotifier

### DISCUSSION THREAD

**Discussion by esDotDev** [rrousselGit/riverpod#866] [Show and tell]
I've created a new example project that has a couple of goals:
* feel familiar to users of provider/change notifier
* focus only on riverpod, no extra concepts (no `freezed`, `hooks`, `stateNotifier` etc)
* show a non-trivial level of app complexity (settings, login etc)

Interested in any feedback! Thoughts in general, or things you think could be better:
https://github.com/esDotDev/flutter_experiments/tree/master/riverpod_todo_with_changenotifier

**Comment by rrousselGit** [comment]
Thanks for that!

If you want to, feel free to make a PR to add it to the list of community examples

**Reply by ferraridamiano** [reply]
Excuse me, where can I find this list of community examples?
I want to learn just riverpod but there is just one very simple example (counter) on this repo. All the others include other concept such as freezed and hooks.
Thank you for all your great work!

**Reply by davidmartos96** [reply]
@ferraridamiano In the online docs there is a section with official and third party examples from the community https://riverpod.dev/docs/getting_started

**Reply by ferraridamiano** [reply]
Thank you @davidmartos96, I didn't see it!
